.gitignore

```diff
@@ -13,4 +13,4 @@ Cargo.lock
 
 .DS_Store
 
-artifacts/checksums_intermediate.txt
+artifacts/
```

Comment:
seems like we don't need the artifacts folder anymore

Reply:
what. why? It is an output folder for the build process and we better gitignore it.

Reply:
no we need it to exclude it from git tracking

Reply:
ah, alright, you're right